src/components/custom/other-platforms.js

```diff
@@ -531,8 +531,9 @@ export const NavCompany = ({ onClick }) => (
         <CardLink
             icon={() => <img src={DerivLife} alt="" width="24" height="24" />}
             title={<Localize translate_text="Deriv life" />}
-            onClick={onClick}
-            to={deriv_life_url}
+            to=""
```

Comment:
I think this is optional and can be left an assigned.

Reply:
I though the same thing, but when I didn't provide the empty string in `to` prop my link would be something like : `https://derivlife.com/undefined`, this is caused by not providing default value for `to` param in `getURLFormat` in `src/components/localization/localized-link.js`, so Unfortunately I have to provide the empty string here.

Reply:
Alright. This will be handled once we refactor this file to typescript.